tests/dbos.test.ts

```diff
@@ -37,6 +37,27 @@ describe("dbos-tests", () => {
     expect(JSON.parse(workflowResult)).toEqual({ current_user: username });
   });
 
+  test("simple-workflow-attempts-counter", async () => {
+    const systemDBClient = new Client({
+      user: config.poolConfig.user,
+      port: config.poolConfig.port,
+      host: config.poolConfig.host,
+      password: config.poolConfig.password,
+      database: config.system_database,
+    });
+    try {
+        await systemDBClient.connect();
+        const handle = await testRuntime.startWorkflow(DBOSTestClass).noopWorkflow()
+        for (let i = 0; i < 10; i++) {
+            await testRuntime.startWorkflow(DBOSTestClass, handle.getWorkflowUUID()).noopWorkflow();
+            const result = await systemDBClient.query<{status: string, attempts: number}>(`SELECT status, recovery_attempts as attempts FROM dbos.workflow_status WHERE workflow_uuid=$1`, [handle.getWorkflowUUID()]);
+            expect(result.rows[0].attempts).toBe(String(i + 2));
```

Comment:
+2 because:

- +1 for the first call to `startWorkflow` above
- +1 because i starts at 0